src/app/calendar/page.tsx

```diff
@@ -1,10 +1,11 @@
 import { Metadata } from "next";
 import React from "react";
+import ParcelCalendar from "@/app/calendar/ParcelCalendar";
 
 const CalendarPage: React.FC<{}> = () => {
     return (
         <main>
-            <h1>Calendar Page</h1>
+            <ParcelCalendar />
```

Comment:
> 'ParcelCalendar' cannot be used as a JSX component.

make the Calendar Page `async` itself to get rid of this error :)

Reply:
Done